docs/use-the-network/roaming/roaming.mdx

```diff
@@ -0,0 +1,97 @@
+---
+id: roaming 
+sidebar_label: Helium Roaming Services
+slug: /use-the-network/roaming
+---
+
+# Helium Roaming Services
+
+Helium Roaming Services enables public and private LoRaWAN network providers around the world to leverage Helium’s ubiquitous coverage. 
+
+With over 630K Hotspots in 166 countries and growing, the network sees a massive amount of non-Helium packets. When network providers enter a roaming partnership with Helium, this traffic can be forwarded via the Helium Network to partners, increasing their level of service and coverage. 
+
+Any compatible device can send data to their home network through Helium-compatible Hotspots without the need for device configurations or firmware updates.
+
+## Getting Started
+- **Create an Account**- Contact a Helium Network Advisor to create the initial Helium Roaming Services account (sales@helium.com). Initial setup requires an Organization name, and an email address for the administrator.
```

Comment:
Should we add anything about Net IDs here, so people don't think they can sign up to buy arbitrary traffic?

Reply:
Something like: As part of the initial account creation process, Net ID ownership must be verified.